python/ray/train/tests/test_backend.py

```diff
@@ -106,6 +121,35 @@ def on_shutdown(self, worker_group: WorkerGroup, backend_config: TestConfig):
         pass
 
 
+class MockWorkerGroup(WorkerGroup):
```

Comment:
clarify in the name that this is specific for fake multi-node testing

```suggestion
class MockMultiNodeWorkerGroup(WorkerGroup):
```